src/gst-plugins/commons/kmsenctreebin.c

```diff
@@ -225,6 +246,7 @@ kms_enc_tree_bin_create_encoder_for_caps (KmsEncTreeBin * self,
       gst_element_factory_list_get_elements (GST_ELEMENT_FACTORY_TYPE_ENCODER,
       GST_RANK_NONE);
 
+#if 0
```

Comment:
This is not nice because it will end up using libx264 (which has royalties and license issues in some places) instead of the OpenH264 encoder, when VAAPI libs are not installed.

Reply:
Can't argue with that. But: We probably need a more fine-grained solution for it, because that way it makes it impossible to make use of the `VAAPI` encoders (see comment below)